deployment-opa.yaml

```diff
@@ -1,47 +0,0 @@
-apiVersion: apps/v1
```

Comment:
Isn't it a parasite change?

Reply:
Not sure what you mean? The file is deleted because it's not needed any more - the deployment is now handled by the operator

Reply:
It's not in the description of the PR, so I would have expected it in a separate PR. But I'm fine with having it here.
Damn, I feel as picky as Adam Grare or Jason Frey on my own PRs for ManageIQ :)